package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
this is a nit...but can we mark `react-bootstrap` as a peer dependency instead? i don't see it being a dependency for all components... but most i suppose.